### ISSUE TITLE
Can't open WSL folder

### ISSUE THREAD

**Issue by velhobit** [adobe/brackets#15012]
### Description

Trying to open a folder on WSL and it doesnt work. I use Windows WSL Ubuntu to serve all my projects to make a better control of data and services. The WSL uses a network reference folder with a $ character.
\\wsl$\Ubuntu

### Steps to Reproduce

File>Open File
Paste the WSL path on address bar
Press enter

**Expected behavior:** 
Open Folder

**Actual behavior:**
Show a network error

### Versions
"bracketsVersion": "1.14.1-17752",
"os: "win10 64"
"osLanguage": "pt-BR"

**Comment by erhassettbg** [comment]
Are you still having this issue? I haven't been able to reproduce it.
I'm able to open WSL folders with: 
* Brackets version 1.14.2-17770 
* Windows 10 build 19624.1000 (version 2004)

**Comment by velhobit** [comment]
Yes. I still having this issue. I typed explorer.exe . to get the current folder, try to paste the WSL path on address bar when Open Folder and shows an error:
the network path cant be found
My path is **\\wsl$\Ubuntu\var\www\html**

![image](https://user-images.githubusercontent.com/1556320/81875585-f518f100-9556-11ea-83c3-7a96d458bc2b.png)

Im using version 1909.

Maybe you arent getting this error cause version 2004 has a better support to access WSL folders.
